Bobsar0/Fast-Food-Fast
# #160552938 ft signup frontend implementation

#### What does this PR do?
Consumes signup endpoint upon user signup

#### Description of Task to be completed?
Receives signup credentials from form, validates credentials and performs successful signup

#### How should this be manually tested?
  Clone the  repo and run the following commands:
``` 
  git checkout ft-signup-frontend-implementation-#160552938
 npm install
  npm start
 ```
Navigate to http://localhost:9999/templates/signup.html, submit credentials and hit the signup button.
If your username is unique, and email and password are valid, you should be redirected to a dedicated menu page

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/#160552938
